app/components/DR/CasesStatistics.js

```diff
@@ -102,8 +107,22 @@ class CasesStatistics extends React.Component {
               styles.dateSubtitle,
               { alignSelf: 'center', textAlign: 'center' },
             ]}>
-            {this.getUpdateDate()}
+            {t('label.date_dashboard_label')}
           </Text>
+          <View style={{ alignSelf: 'center' }}>
```

Comment:
it's not necessary to add this `view` to center the calendar, you can add the style in the component, something like this

```
<CalendarButton
              style={{ backgroundColor: '#FFF', alignSelf: 'center' }}
```